staging/src/k8s.io/apiserver/pkg/endpoints/handlers/responsewriters/writers.go

```diff
@@ -109,9 +109,6 @@ func WriteObjectNegotiated(ctx request.Context, s runtime.NegotiatedSerializer,
 		audit.LogResponseObject(ae, object, gv, s)
 	}
 
-	w.Header().Set("Content-Type", serializer.MediaType)
```

Comment:
Thanks for your review.
http.ResponseWriter will log an error if WriteHeader is called multiple times.
https://github.com/golang/go/blob/master/src/net/http/server.go#L1048

But kubernetes has this:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/apiserver/pkg/server/filters/timeout.go#L188

So we didn't see anything when WriteHeader is called multi times.